### DISCUSSION TITLE
Is python updated with sd.next?

### DISCUSSION THREAD

**Discussion by Sunsphire** [vladmandic/sdnext#2127] [Q&A] [answered]
I have an absolute newbie, probably stupid question...Should I upgrade or care about upgrading Python 3.10.9 to Python 3.10.10/3.10.11?

I have a total brain fart and I think it's FOMO. I saw that lots of people have Python 3.10.10 or 3.10.11 (on Issues tab for example). That might be related to their hardware, but I'm not sure.
I know from looking into other discussions that 3.10.3, 3.10.11...or whatever 3.10.X work, but is there any benefit upgrading from 3.10.9 to 3.10.10 or 3.10.11?

In case it's related to the hardware/OS:
Windows 10, rtx 2080, 5950X

**Comment by vladmandic** [comment, marked as answer]
in general, its always best to be on the latest patch level (which is what 3rd digit means).
but is it critical or necessary? no.
if you do have issues and you're on old patch release, its always best to upgrade to latest patch level first to see if issue is resolved. but if you don't have issues, its not going to work any better.

**Reply by Sunsphire** [reply]
That's exactly what I wanted to know. Thank you for your fast reply and amazing work!
